create-keycloak-realm/variables.tf

```diff
@@ -28,4 +28,10 @@ variable "tenant_id" {
 
 variable "api_version_path" {
   type = string
+}
+
+variable "deploy_automation_client" {
+  type        = bool
+  default     = false
```

Comment:
I think the default value here is not strictly necessary since we only use this sub-module from its parent and by always passing it a value (the default value being at the whole parent module level).